### PR TITLE
Adding ColorTable as component

### PR DESCRIPTION
**Pull request by elenagiere**
This is to showcase all of the Momentum semantic color tokens in a clean table.

## Description
Renders a table showing all of the momentum semantic color tokens for light and dark mode.

Screen Shot 2021-03-16 at 6 57 35 PM

## Screenshots:
**Before** (If applicable):

**After**:

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation update
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other... Please describe:

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: web-components/src/[sandbox]/sandbox.ts [hunk 1]
```diff
@@ -213,6 +214,13 @@ export class Sandbox extends LitElement {
             </sass-stats>
           </div>
 
+          <div class="container" aria-label="md-colors">
+            <h2>md-colors</h2>
+            <sass-stats component="table">
```
Comment (yananym) [inline web-components/src/[sandbox]/sandbox.ts:219]: do we really need sass stats for this?

File: web-components/src/components/color-table/ColorTable.ts [hunk 2]
```diff
@@ -0,0 +1,85 @@
+import "@/components/table-advanced/TableAdvanced";
```
Comment (yananym) [inline web-components/src/components/color-table/ColorTable.ts:1]: This file is in the components folder - but it is not a reusable component, it is for Storybook only... We should remove it from the components folder

**Review by yananym — CHANGES_REQUESTED** [review]

File: web-components/src/[sandbox]/sandbox.ts (hunk 1 above)
Comment (yananym) [reply web-components/src/[sandbox]/sandbox.ts:219]: Or at least add it to another folder (e.g. Internal/Not Exportable...) or give the folder a prefix... something

File: web-components/src/components/color-table/ColorTable.ts (hunk 2 above)
Comment (yananym) [reply web-components/src/components/color-table/ColorTable.ts:1]: Or at least add it to another folder (e.g. Internal/Not Exportable...) or give the folder a prefix... something

File: web-components/src/[sandbox]/sandbox.ts (hunk 1 above)
Comment (futurethang) [reply web-components/src/[sandbox]/sandbox.ts:219]: Nope

File: web-components/src/components/color-table/ColorTable.ts (hunk 2 above)
Comment (elenagiere) [reply web-components/src/components/color-table/ColorTable.ts:1]: I did this because it didn't seem to load scss well otherwise. I can try.

Comment (elenagiere) [reply web-components/src/components/color-table/ColorTable.ts]: I have created an `internal-components` folder separate from components. Not it is displayed within a storybook folder called `internal references`.

File: web-components/src/components/table-advanced/TableAdvanced.ts [hunk 3]
```diff
@@ -707,6 +707,7 @@ export namespace TableAdvanced {
 
         const groups = rows.reduce((acc, row) => {
           const key = row.cells[colIdx].text;
+          //console.log(row.cells);
```
Comment (yananym) [inline web-components/src/components/table-advanced/TableAdvanced.ts:710]:
```suggestion
```

**Review by yananym — DISMISSED** [review]

**Pull request merged**